app/assets/javascripts/govuk_publishing_components/analytics-ga4/pii-remover.js

```diff
@@ -32,6 +32,10 @@
   var VISA_PATTERN_GWF = /GWF\d{9}/g
   var VISA_PATTERN_GB = /GB\d{12}/g
 
+  // 2 valid letters - optional space - 6 digits with optional spaces every 2 digits (to account for '12 34 56') - optional space - 1 valid letter
+  // e.g. AB123456A or AB 12 34 56 A
+  var NATIONAL_INSURANCE_NUMBER = /[A-CEGHJ-OPR-TW-Z]{2}\s?(\d{2}\s?){3}\s?[A-D]/gi
```

Comment:
Will this be used to redact NI numbers from URLs? If you type a national insurance number into the GOV.UK search the URL comes out with `+` characters instead of spaces e.g. `https://www.gov.uk/search/all?keywords=BR+12+34+56+` - will this still be redacted?